supreme_court_predictions/models/model.py

```diff
@@ -21,18 +23,25 @@ def create(self, df):
         :return: A sklearn model.
         """
 
-    def create_and_measure(self, df, accuracy_measure):
+    def create_and_measure(self, df):
         """
         Takes in a dataframe and returns the applicable accuracy measurement.
 
         :param pandas.DataFrame df: A dataframe used to create the model.
         :param function accuracy_measure: A function that is used to measure
         accuracy on the model.
-        :return: Float of some accuracy measurement.
+        :return: Float of accuracy.
+        :return: Float of F1 score.
+        :return: Confusion matrix for model
         """
-        _, y_test, y_pred = self.create(df)
+        model, y_test, y_pred = self.create(df)
 
-        return accuracy_measure(y_true=y_test, y_pred=y_pred)
+        return (
+            model,
+            accuracy_score(y_true=y_test, y_pred=y_pred),
+            f1_score(y_true=y_test, y_pred=y_pred),
+            confusion_matrix(y_true=y_test, y_pred=y_pred),
```

Comment:
Are these three functions the same for all models?

Reply:
Yup, they’re general sklearn functions that aren’t model dependent.